src/SFML/Graphics/Font.cpp

```diff
@@ -152,9 +152,9 @@ const sfTexture* sfFont_getTexture(sfFont* font, unsigned int characterSize)
 {
     CSFML_CHECK_RETURN(font, nullptr);
 
-    *font->Textures[characterSize].This = font->This.getTexture(characterSize);
+    font->Textures.emplace(std::make_pair(characterSize, sfTexture(new sf::Texture(font->This.getTexture(characterSize)))));
```

Comment:
This is not the same. I suggest you try to get existing sfTexture with map::find. If found, assign underlying sf::Texture as before, if not found use map::emplace but without make_pair to make sure the sfTexture is constructed in place and not copied from a temporary pair (because copying an owning sfTexture is an ownership error).